upload/catalog/model/localisation/currency.php

```diff
@@ -28,7 +28,7 @@ public function editValueByCode(string $code, float $value): void {
 	 * @return array<string, mixed>
 	 */
 	public function getCurrency(int $currency_id): array {
-		$query = $this->db->query("SELECT DISTINCT * FROM `" . DB_PREFIX . "currency` WHERE `currency_id` = '" . $this->db->escape($currency_id) . "'");
+		$query = $this->db->query("SELECT DISTINCT * FROM `" . DB_PREFIX . "currency` WHERE `currency_id` = '" . (int)$currency_id . "'");
```

Comment:
This was already cast to an in in the function signature so doesn't even need the explicit cast here.